client/src/components/SentimentAnalysis/SimilarAds.js

```diff
@@ -0,0 +1,59 @@
+import { database } from '../../firebase/firebase';
+import React, { useState, useEffect } from 'react';
+
+const SimilarAds = props => {
+
+  const COLLECTION = 'ads';  
+  const headlineSentiment = `{"score": ${props.headlineScore}, "magnitude": ${props.headlineMagnitude}}`;
+  const contentSentiment = `{"score": ${props.contentScore}, "magnitude": ${props.contentMagnitude}}`;
+
+  const [results, setResults] = useState([]);
+  let resultItems = [];
+
+  useEffect(() => {
+    async function fetchSimilarAds() {
+      
+      let similarAdsPromises = [];
+      similarAdsPromises.push(database.collection(COLLECTION)
+            .where('headlineSentiment', '==', headlineSentiment)
+            .where('contentSentiment', '==', contentSentiment)
+            .limit(10)
+            .get()
+      ); 
+      let similarAds = await Promise.all(similarAdsPromises);
```

Comment:
Since there is only one promise being awaited you don't really need `Promise.all`.